moviesearch/search.py

```diff
@@ -0,0 +1,157 @@
+from BeautifulSoup import BeautifulSoup
+from mechanize import Browser
+import re
+import json
+
+
+def get_soup(movie):
+    movie = '+'.join(movie.split())
+    url = "http://www.imdb.com/find?ref_=nv_sr_fn&q="+movie+"&s=all"
+    br = Browser()
+    br.open(url)
+    try:
+        link = list(br.links(url_regex=re.compile(r"/title/tt*")))[0]
+    except:
+        return ""
+    else:
+        res = br.follow_link(link)
+        soup = BeautifulSoup(res.read())
+        return soup
+
+
+def get_title(movie, soup):
+    try:
+        return soup.find('title').contents[0]
+    except:
+        return "N/A"
```

Comment:
if you want to return the same thing from every function, use a constant perhaps?